tests/Avalonia.Controls.UnitTests/TextBlockTests.cs

```diff
@@ -498,7 +496,7 @@ public void TextBlock_With_UseLayoutRounding_True_Should_Round_DesiredSize()
 
             target.Measure(new Size(double.PositiveInfinity, double.PositiveInfinity));
 
-            Assert.Equal(target.DesiredSize, new Size(40, 10));
+            Assert.Equal(target.DesiredSize, new Size(28, 15));
```

Comment:
It was already the case before this PR, but all those modified `Assert.Equal` have the parameters reversed. The expected one should be first.